app/views/users/show.html.erb

```diff
@@ -1,7 +1,14 @@
-<h1><%= @user.name %></h1>
+
+<div class="show-user">
+<div class="container"><div class="cadre"></div><div class="user"><h2><%= @user.name  %></h2> <p><span></span><span>Number of posts : <span><%= @user.posts_counter %></span></span></p></div></div>
+
+<p class='bio'><%= @user.bio %></p>
 <br>
-<p><%= @user.bio %></p>
+<% @post.each_with_index do |posts, idx|%>
+    <% break if idx == 3 %>
+    <div class='post'><span class='title'><%= link_to 'Post #', user_posts_path(@user) %><%= idx + 1 %></span>
+    <br><%= posts.text %></div>
+<% end %>
 <br>
-<%= @user.posts_counter %>
-<br>
-<%= link_to 'Go to Posts', users_path %>
```

Comment:
- [ ] According to the previous reviewer, please consider adding a link to `Post#index` that shows all user's posts. 🤓

Reply:
Hello reviewer, could you enlighten me